src/components/SearchJobs.js

```diff
@@ -1,30 +1,59 @@
-
 import React, {Component} from 'react'
 import uniq from 'lodash/uniq'
 import includes from 'lodash/includes'
 import JobFetcher from './JobFetcher'
 import JobList from './JobList'
 
+const JOB_TYPES = {
+  internship: 'Internship',
+  fulltime: 'Full time',
+  parttime: 'Part time',
+  contract: 'Contract',
+  casual: 'Casual',
+  unknown: 'Unknown'
+}
 
-
-
-class FilterJobs extends Component{
+class FilterJobs extends Component {
   constructor() {
     super()
     this.state = {
       search: "",
-      work_types: {unknown: true, internship: true, fulltime: true, parttime: true, casual: true, contract: true},
+      workTypes: Object.keys(JOB_TYPES).reduce((acc, val) => {acc[val] = true; return acc}, {}),
       suburb: "All suburbs"
     }
   }
```

Comment:
pulled out event handlers into their own functions to both

* remove app logic from `render()`
* make them more visible and amenable to change